Common/Tools/CMakeLists.txt

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
This change is not needed

Reply:
This is not meant to be there and removed now.